projects/packages/search/src/customberg/lib/wordpress-interface.scss

```
@@ -1,11 +1,11 @@
-// These styles are copied directly from @wordpress/interface
-// in order to remove our dependency on this experimental package.
-
-// TODO: Replace these styles with our own styles.
-
-//
-// complementary-area-header
-//
+/**
+ * These styles are copied directly from @wordpress/interface
+ * in order to remove our dependency on this experimental package.
+ *
+ * TODO: Replace these styles with our own styles.
+ *
+ * complementary-area-header
+ */
```

Semantically this seems like it should be more like this
```suggestion
/**
 * These styles are copied directly from @wordpress/interface
 * in order to remove our dependency on this experimental package.
 *
 * TODO: Replace these styles with our own styles.
 */

// complementary-area-header
```